cypress/objects/timeLogingPage.js

```diff
@@ -0,0 +1,27 @@
+class TimeLogingPage {
+    getUrl(){
+        return cy.url().should('include', '/time-logging')
+    }
+    getPageTitle(){
+        return cy.get('.page__title')
+    }
+    getCalendarVisible(){
+        return cy.get('.calendar')
+    }
+    getCalendarToday(){
+        return cy.get('.calendar--today')
+    }
+    getTileForm(){
+        return cy.get('.tile.form')
+    }
+    getInfoTitle(){
+        return cy.get('.user-info__title')
+    }
+    getLength(){
+        return cy.get('.main-nav')
+    }
+    getTimeLoggingColor(){
```

Comment:
This does not retrieve the color of the Time Logging tab, it only returns the active tab, so this should probably be renamed to something like `getActiveTab()`